src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -8,136 +8,125 @@
        i18n>Configuration</a>
     <ng-template ngbNavContent>
       <div>
-        <cd-alert-panel
-          *ngIf="!rgwModuleStatus"
-          type="info"
-          spacingClass="mb-3"
-          class="d-flex align-items-center"
-          i18n
-          >In order to access the import/export feature, the rgw module must be enabled
-
+        <cd-alert-panel *ngIf="!rgwModuleStatus"
+                        type="info"
+                        spacingClass="mb-3"
+                        class="d-flex align-items-center"
+                        i18n>In order to access the import/export feature, the rgw module must be enabled
```

Comment:
is this just some lint changes? doesn't seem any change to code.

Reply:
This is just some linting adjustment, no code change